src/main/java/liuni/TwitterStatus.java

```diff
@@ -11,33 +10,23 @@
 
     private static final int TWITTER_CHAR_MAX = 280;
 
-    public void postStatus(String text) {
+    public void postStatus(Object text) throws Exception {
```

Comment:
shouldn't this be a String instead of Object?

Reply:
I thought you said to allow it to accept multiple inputs?

Reply:
let's ignore the multiple inputs for now

Reply:
Kay Ill revert it